_component_design/actions-menu.md

```diff
@@ -18,27 +17,45 @@ helix-ui-javascript: true
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### When to use
+## When to use
 
-- Use actions menus to consolidate multiple related actions into a single menu.
+An actions menu consolidates multiple related actions into a single menu. Use an actions menu in the following situations:
 
-### When not to use
+  - When a user takes action on the current page or in a [modal]({{site.baseurl}}/components/modals.html)
+  - The user creates or edits items on a different page
+  - The user starts a workflow
 
-- If a table object has only a limited number of associated actions (such as create and delete), use table controls at the top of the table. For more information, see the [tables]({{site.baseurl}}/components/tables.html) documentation.
+Do not use an actions menu for objects with a small number of actions. For example, do not use an actions menu for table objects that have only a create and delete action. When a table object has small number of actions, use table controls at the top of the table. For more information, see [tables]({{site.baseurl}}/components/tables.html).
 
 {% endcolumn %}
 
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+</div>
 
-{% figure [caption:"Example actions menus from various triggers"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/components/inputs-and-controls/actions-menu/actions-menu-hero.png" width="469"/>
-{% endfigure %}
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct an actions menu.
+
+- Items in the actions menu must initiate an action. Do not use an actions menu for navigation.
+- If an action name is longer than the length of the button, extend the menu beyond the right side of the button.
+- If an action label is too long, truncate the text and display the entire name when the users hover over the label.
+
+Consider the following best practices when you construct an actions menu that contains many items.
+
+- Group related items into headers. Headers provide clarity and help users locate menu items of interest.
+- Use a header when the actions menu includes more than three actions. Do **not** use a header when the actions menu includes three or fewer actions.
```

Comment:
"Use a header only when the actions menu includes more than three actions."

The "do not" sentence is redundant.